skietbaan_fe/src/components/Login.js

```diff
@@ -1,98 +1,135 @@
 import React, { Component } from 'react';
 import {
-  Container, Col, Form,
-  FormGroup, Label, Input,
-  Button,
+  Container, Col,
+  FormGroup, Label, Input,Button,Form
 } from 'reactstrap';
 import '../components/LoginStyles.css';
 
+  function validatePassword(str) {
```

Comment:
You can put your 3 functions in 3 separates files that export them.
You will have to import them as modules.

Reply:
Could i create one validate file and store all of those in there

Reply:
That should work. Yes.